phpBB/adm/style/acp_storage_update_inprogress.html

```diff
@@ -0,0 +1,32 @@
+{% include 'overall_header.html' %}
+
+<a id="maincontent"></a>
+
+<h1>{{ lang('STORAGE_TITLE') }}</h1>
+
+<script>
+	// <![CDATA[
+	/**
+	 * Popup storage update progress bar
+	 */
+	function popup_progress_bar()
```

Comment:
Does this still work? I thought we modified the progress bars to not use this popup style anymore, at least for search index.